lib/perl/Genome/Config/AnalysisProject/Command/AddConfigFile.pm

```diff
@@ -70,4 +79,28 @@ sub _mark_instrument_data_bridges {
     return 1;
 }
 
+sub _apply_tags {
+    my ($self, $profile_item) = @_;
+    return unless $self->tag;
+    for my $tag_name ($self->tag){
+        my $tag = Genome::Config::Tag->create(name => $tag_name);
+        $profile_item->add_tag($tag);
+    }
+    return 1;
+}
+
+sub __errors__ {
```

Comment:
Did you mean to add `__errors__` in this commit? (The base class should take care of this.)